src/handler/policies/fetchTokenInformation.ts

```diff
@@ -0,0 +1,46 @@
+import { GuardianMiddlewareRequest } from 'src/context/useGuardianContext'
+import Response from 'src/response'
+import { NextApiResponse } from 'next'
+import { Tag } from 'src/config/guardianTags'
+
+async function FetchTokenInformation(
+	req: GuardianMiddlewareRequest,
+	res: NextApiResponse
+) {
+	const { accessToken } = req
+	const { policyId } = req.query
+	const { guardian } = req.context
+
+	const data = await guardian.policies.policyById(
+		accessToken,
+		String(policyId)
+	)
+
+	const mintTokenPath = [
+		Tag.verifierWorkflow,
+		Tag.approveIssueRequestsPage,
+		Tag.mintTokenParent,
+		Tag.mintToken,
+	]
+
+	// 😅 I feel Justyn already wrote this...
```

Comment:
Good memory! Not in this repo but we wrote something similar here https://github.com/dovuofficial/guardian-stories/blob/f79466abd5f34045d7b15545cd17ace6d38cf6b3/scripts/utilities/policy-config-updater.js#L26

Reply:
Indeed reduce is better here.